packages/react-metaforms-antd/src/components/Multiselect.tsx

```diff
@@ -73,9 +70,7 @@ const Multiselect = React.forwardRef(({ options = [], value = [], onChange, plac
   };
 
   const handleSelectAll = (checked: boolean) => {
-    const optValues = checked ? options.map(opt => opt.value) : [];
-    setFilteredOptions(checked ? options : []);
```

Comment:
Tohle jsi asi omylem smazal. Je to tu kvůli přepínání mezi zobrazením jen vybraných options a všech